python/lsst/daf/butler/script/_associate.py

```diff
@@ -18,13 +18,23 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
+from __future__ import annotations
+
+from typing import Iterable
```

Comment:
Should it be `collections.abc` instead?

Reply:
Thanks. I forgot that I'm supposed to be using those directly now rather than `typing.Iterable`.